Cargo.toml

```diff
@@ -17,16 +17,19 @@ keywords = ["parity", "substrate", "blockchain"]
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
 
 [features]
-default = []
+default = ["tokio1"]
 client = ["substrate-subxt-client"]
+# jsonrpsee http client can be configured to use tokio02 or tokio1.
+tokio02 = ["jsonrpsee-http-client/tokio02"]
+tokio1 = ["jsonrpsee-http-client/tokio1"]
 
 [dependencies]
 log = "0.4.14"
 thiserror = "1.0.24"
 futures = "0.3.13"
 jsonrpsee-types = "=0.2.0-alpha.3"
 jsonrpsee-ws-client = "=0.2.0-alpha.3"
-jsonrpsee-http-client = "=0.2.0-alpha.3"
+jsonrpsee-http-client = { version = "=0.2.0-alpha.3", default-features = false, optional = true }
```

Comment:
Why is it `optional`, in case we use `default-features  = false` when depending on subxt. Wouldn't that cause a compilation error?

Reply:
right good catch, it doesn't need to be optional.

However, if tokio1 or tokio02 is not enabled on jsonrpsee it will be a compilation error anyway. So doesn't really make a difference in practice.